### PR TITLE
Add welcome_card for auto-generated dashboard

### PR DESCRIPTION
**Pull request by basbruss**
Add `card_welcome_scenes`
Add translations to main files
image

### CODE REVIEW

**Comment by rick2909** [comment]
it looks liike it is the same one that is now a custom one, maybe remove the custom card.

**Comment by basbruss** [comment]
@stokkie90 could you test these changes in your dev-env? My system had some hiccups with loading of javascript and the templates, but the code should be correct.

**Comment by AndyVRD** [comment]
Thanks for the merge and release!

**Comment by basbruss** [comment]
Formated date chip to:
image

**Comment by stokkie90** [comment]
@basbruss i tested it on my dev instance:

- After installation, first time you will get, can't find the cards.  (caching etc) but after ctrl + f5 and/or refresh (top-right-corner) it load the dashboard.
- After installation no restart is required. but it will not load themes. so we might need to extend docs a bit. with the action to restart or reload themes. using the commands  (hit "c" on keyboard)

File: custom_components/ui_lovelace_minimalist/lovelace/ui-lovelace.yaml [hunk 1]
```diff
@@ -29,10 +29,14 @@ views:
...
                   options:
                     type: "custom:button-card"
-                    template: "card_paddy_welcome_with_weather"
+                    template: "card_welcome_scenes"
                     variables:
-                      ulm_card_paddy_welcome_time: "sensor.time"
-                      ulm_card_paddy_welcome_weather_provider: "this.entity_id"
+                      ulm_time: "sensor.time"
+                      ulm_weather: "this.entity_id"
+          - type: "custom:button-card"
```
Comment (stokkie90) [inline custom_components/ui_lovelace_minimalist/lovelace/ui-lovelace.yaml:36]: Double title "Lights"

**Review by stokkie90 — APPROVED** [review]

**Pull request merged**